byway/src/main/webapp/script.js

```diff
@@ -12,17 +12,38 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+var selected = [];
```

Comment:
Generally speaking, it is almost always better to declare varaibles with `let` or `const` instead of `var`. The difference is that `var` is globally scope. This can lead to lots of confusing issues. You can read more about this here: https://www.javascripttutorial.net/es6/difference-between-var-and-let/

 Now that everyone has gotten a decent handle on javascript, lets switch to using `let` in place of `var`.

Reply:
Fixed